sdk/tests/conformance2/extensions/00_test_list.txt

```diff
@@ -1,6 +1,7 @@
 ext-color-buffer-float.html
 ext-disjoint-timer-query-webgl2.html
 --min-version 2.0.1 ext-texture-filter-anisotropic.html
+--min-version 2.0.1 ext-texture-norm16.html
```

Comment:
Expose to WebGL 1

Reply:
Per discussion in WebGL WG today, let's leave this as a WebGL 2.0-specific extension for the initial iteration.